lib/Textile/Api.ts

```diff
@@ -97,12 +105,14 @@ export async function avatar(): Promise<string | undefined> {
 
 export async function cafeSession(peerId: string): Promise<CafeSession> {
   const result = await TextileNode.cafeSession(peerId)
-  return JSON.parse(result) as CafeSession
+  const buffer = Buffer.from(result, 'base64')
+  return JSON.parse(buffer.toString()) as CafeSession
```

Comment:
Pretty sure this and the other session related methods need to use the generated JS protobuf types to parse this, not JSON.parse.

Reply:
oh, yeah, looks that way... I had figured you were just marshaling to JSON because you had said no need to worry about the JS protobufs

Reply:
ah, i see... they are already generated. cool, will update.